team-manager-api/src/main/java/ua/knu/knudev/teammanagerapi/dto/RecruitmentOpenRequestDto.java

```diff
@@ -0,0 +1,13 @@
+package ua.knu.knudev.teammanagerapi.dto;
+
+import lombok.Builder;
+import lombok.Data;
+import ua.knu.knudev.knudevcommon.constant.Expertise;
+
+@Data
+@Builder
+public class RecruitmentOpenRequestDto {
+
+    private String recruitmentName;
+    private Expertise expertise;
+}
```

Comment:
Add here nested record called `AutoCloseExceeding` and add there:
int maximumRecruitedPeople, LocalDateTime autoClosingDate

So, it will be:
`
public record RecruitmentOpenRequest(String recruitmentName, Expertise recruitmentExpertise, AutoCloseExceeding autoCloseExceeding) {
    private record AutoCloseExceeding(int recruitedPeopleLimit, LocalDateTime autoClosingDate) {}
}
`